test/e2e/kubernetes/windows/windows.go

```diff
@@ -52,7 +52,7 @@ func (i *WindowsImage) GetImage(version OSVersion) string {
 
 func GetWindowsImage(testImage WindowsTestImages, version OSVersion) string {
 	httpd := WindowsImage{
-		BaseImage: "k8sprow.azurecr.io/kubernetes-e2e-test-images/httpd",
+		BaseImage: "registry.k8s.io/e2e-test-images/httpd",
 		Tags: map[OSVersion]string{
 			LTSC2019: "2.4.39-alpine",
```

Comment:
let's bump this version here otherwise we get the error: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/3273/pull-cluster-api-provider-azure-e2e/1634332087173517312/artifacts/clusters/capz-e2e-ankuc9-ha/default/web-windowssl75kv-855795778f-mrjj8/web-windowssl75kv.log

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3262#discussion_r1132752919

Reply:
lets use the latest `registry.k8s.io/e2e-test-images/httpd:2.4.39-4`

Reply:
looks like a fix when in later https://github.com/kubernetes/kubernetes/pull/99030